### PR TITLE
Implement separate files for global errors

### PR DESCRIPTION
**Pull request by meshnesh**
# What does this PR do?

Implement global error handling, Implement DRY principles in the code

## Description of tasks to be completed
Create a file to hold the error handles, import the errors to the app
Create decorator to handle common code samples to avoid repetition

## How should this be tested manually?

Open a url with a `GET` request change it to a `DELETE` request to view the error.
In a url delete a letter or word and press enter to view the 404 error.
Delete any value required by the server to get a 500 internal error.

### CODE REVIEW

**Review by Rodgers-M — COMMENTED** [review]
nice work on the error handlers.

File: app/errors/error.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+"""import depancies and methods."""
+
+from flask import make_response, jsonify
+
+def create_error_handler(app, errorno):
+    @app.errorhandler(errorno)
+    def handler(error):
+        return
```
Comment (frankip) [inline app/errors/error.py:8]: no return value or is it supposed to be like that?

File: app/errors/error.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    return handler
+
+
+def not_found_error(error):
+    response_object = {
+        'message': 'Object not found'
+    }
+    print('manenos')
```
Comment (frankip) [inline app/errors/error.py:16]: get rid of the print statements

**Review by frankip — COMMENTED** [review]
good implementation

File: app/errors/error.py (hunk 2 above)
Comment (meshnesh) [reply app/errors/error.py:16]: thanks

File: app/errors/error.py (hunk 1 above)
Comment (meshnesh) [reply app/errors/error.py:8]: yes it fine like that

**Pull request merged**